### ISSUE TITLE
Accessibility issue: Required aria attributes must be provided on /react/storybook/

### ISSUE THREAD

**Issue by lindseywild** [lindseywild/accessibility-scanner#8]
## What
An accessibility scan flagged the element `<button aria-label="Measure tool" role="switch" class="css-h6cg0q" tabindex="0">` on https://primer.style/react/storybook/?path=/story/components-banner-features--actions-layout-inline # Buttons are overlapping because required aria attributes must be provided. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/aria-required-attr?application=playwright.

To fix this, ensure elements with aria roles have all required aria attributes.

Specifically:

Fix any of the following:
-   Required ARIA attribute not present: aria-checked

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
